cmd/options/controller_options.go

```diff
@@ -33,10 +33,13 @@ type ControllerOptions struct {
 	ExtraVolumeTags map[string]string
 	// ID of the kubernetes cluster.
 	KubernetesClusterID string
+	// AWS Assume Role ARN
+	AWSAssumeRole string
```

Comment:
Let's rename this to  `AssumeRoleArn`to make the "arn" requirement more explicit.